app/assets/javascripts/covid19/src/data/repositories/Covid19InfoFromJsonRepository.ts

```diff
@@ -102,19 +102,18 @@ export class Covid19InfoFromJsonRepository implements Covid19InfoRepository {
         matches: string[] = []
     ): Structure[] {
         const miniSearch = this.getMiniSearch();
-        const searchOptions =
-            matches.length > 0
-                ? {
-                      ...this.searchOptions,
-                      filter: (result: SearchResult) =>
-                          _.isEmpty(
-                              _.difference(
-                                  matches.map(m => m.toLowerCase()),
-                                  result.terms.map(m => m.toLowerCase())
-                              )
-                          ),
-                  }
-                : this.searchOptions;
+        const searchOptions = !_.isEmpty(matches.length)
```

Comment:
`!_.isEmpty(matches)` ?

Reply:
Thank you for your thorough review, I rushed that change without much attention, I'm ashamed 😓